### DISCUSSION TITLE
Show Group name of 'Group Column' option offered in Value Relation Widget in QField

### DISCUSSION THREAD

**Discussion by Meykaefer** [opengisch/QField#5480] [Q&A]
Hey everyone,

The recent update of QGIS v.3.38 introduced the possibility in the Value Relation Widget to group items via the  'Group Column' option and a field value. Below is a checkbox that says 'Display group name'. It works great in QGIS on Desktop, but when I transferred my project to my iPad, I noticed that the 'Group column' option works well, but the group name does not get displayed despite having the 'Display group name' checked. Is this not supported yet, or am I missing a setting somewhere?

I love the functionality, but without seeing the group name in QField, it is less useful than it could be.

**Comment by nirvn** [comment]
Are you running on an old QField version? Or maybe you misunderstand the setting. When enabling display group name, it means the group will be displayed in the combo box, like this:

![image](https://github.com/user-attachments/assets/ed888d7e-6a07-435b-a9f6-42ccd280bbf3)

**Reply by Meykaefer** [reply]
I am currently running 3.3.7 - Darien, which should be the most up-to-date version in the App Store.

Mine didn't look like your screenshot, so I did some testing. Initially, I had "Allow multiple selections" checked and didn't get the result shown in your screenshot. I unchecked it once, tried it, and got the result in your screenshot above.

I am not 100% sure if I misunderstood the setting. It works well on the desktop version of QGIS with "Allow multiple selections" but doesn't work in QField.

**Reply by nirvn** [reply]
@Meykaefer , ahh, I see, it's indeed an implementation gap.

**Reply by nirvn** [reply]
@Meykaefer , fixed; you can test that it works for you using APKs here: https://github.com/opengisch/QField/pull/5482

**Reply by Meykaefer** [reply]
@nirvn I tested it on the iPad after the QField v3.8 went live in the App Store. Works perfectly! Thank you for fixing it so quickly!
